cpp/src/copying/concatenate.cu

```diff
@@ -428,21 +428,21 @@ std::unique_ptr<table> concatenate(std::vector<table_view> const& tables_to_conc
   std::vector<std::unique_ptr<column>> concat_columns;
   for (size_type i = 0; i < first_table.num_columns(); ++i) {
     std::vector<column_view> cols;
-    std::transform(tables_to_concat.cbegin(),
-                   tables_to_concat.cend(),
+    std::transform(tables_to_concat.begin(),
+                   tables_to_concat.end(),
                    std::back_inserter(cols),
                    [i](auto const& t) { return t.column(i); });
 
     // verify all types match and that we won't overflow size_type in output size
     bounds_and_type_check(cols.begin(), cols.end());
-    concat_columns.emplace_back(detail::concatenate(cols, stream, mr));
+    concat_columns.emplace_back(detail::concatenate(host_span<column_view>{cols}, stream, mr));
```

Comment:
Why do we need to explicitly convert here?

Reply:
Looks like I missed that one. Thanks.